README.md

```diff
@@ -7,6 +7,28 @@ It will listen to audio input and produce mesmerizing visuals. Some commands are
 This project is in a bit of a transition state and is in the process of being modernized. There are many rough edges at
 present.
 
+## GUI controls
+
+
+| Command | Keyboard | Mouse | Game Controller |
+| --- | --- | --- | --- |
+| Add a new waveform | | `ctrl` + Left |  |
+| Clear all custom waveforms | | Middle |  |
```

Comment:
This functionality was not reimplemented when rewriting the renderer, as it doesn't really fit the structure well and was quite cumbersome to use anyway. No need to add controls for something that doesn't work.
```suggestion
```